stable/firehose/templates/deployment.yaml

```diff
@@ -85,3 +85,11 @@ spec:
       - emptyDir: {}
         name: workdir
       {{- end }}
+      {{- if index .Values "taints" "enabled" }}
+      tolerations: {{- range $key, $value := .Values.taints.list }}
```

Comment:
are we going to do the sink-type based selection in dex itself? what i (and @AkarshSatija ) had in mind was doing that switch here.. (i.e., if sink_type = log, choose the tolerations set under `taints.log` etc.)

Reply:
I also have this question, if we need to do this filtering.
If yes, 
- Do we need to check if the `firehose.config.SINK_TYPE` is one of `log` or `bigquery`, then we shall find the taint config for it and put it in the pod config? 
- Do we need to allow this for only the above two sink types?

Reply:
This chart should not have context about sink_type logic, it should only care about if there is a toleration or affinity needed.

The logic of checking which sink type should  have tolerations ideally would be in entity or dex.

From helm chart perspective, we could a new variables with firehose k8s deployment context only. We could have new variables called `tolerations` and `affinity` since those are the terms that the deployment manifest understands.

Reply:
this is done